webserver/webserver/src/main/java/io/helidon/webserver/NettyWebServer.java

```diff
@@ -163,6 +163,9 @@ class NettyWebServer implements WebServer {
                      .childHandler(childHandler);
 
             bootstraps.put(name, bootstrap);
+
+            // subscribe to updates
+            soConfig.tls().ifPresent(tlsConfig -> updateTls(tlsConfig, name));
```

Comment:
This does not subscribe to updates. This actually registers yet another tls and overwrites the one created few lines above :-)

Reply:
See last commit.